packages/main/src/types/RatingIndicatorSize.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Different types of RatingIndicatorSize.
+ * @public
+ */
+enum RatingIndicatorSize {
+	/**
+	 * icon size - 1.375rem
```

Comment:
Don't mention fixed sizes here because it might result in something different depending on the theme. Also add some more meaningful description to the whole enum.

Reply:
add also since 2.6.0 tag